docs/TESTED.md

```diff
@@ -28,17 +28,17 @@ To aid users in troubleshooting, we've compiled a list of users who've reported
 | ^^ | ^^ | webg3 | Stock Model |
 | MacBookAir5,2 | ^^ | Bab-droid | - Stock Model<br/>- Patcher version 0.0.19 |
 | ^^ | ^^ | K-Hobert | - Stock model <br/>- Patcher version unknown |
-| ^^ | ^^ | cboukouv |  - Stock model <br/>- Patcher version 0.0.21 |
+| ^^ | ^^ | cboukouv | - Stock model <br/>- Patcher version 0.0.21 |
 
 ### MacBook Pro
 
 | SMBIOS | Tested | Tester | Comment |
 | :--- | :--- | :--- | :--- |
-| MacBookPro4,1 | <span style="color:#30BCD5"> YES </span> | cboukouv | - Stock model <br/>- Patcher version 0.0.19  |
+| MacBookPro4,1 | <span style="color:#30BCD5"> YES </span> | cboukouv | - Stock model <br/>- Patcher version 0.0.19 |
 | MacBookPro5,1 | ^^ | GURU_Jasho#5736 | - Stock model <br/>- Patcher version 0.0.21 |
-| MacBookPro5,2 |  ^^ | hvds | ^^ |
-| MacBookPro5,3 | No reports | N/A | N/A |
-| MacBookPro5,4 | ^^ | ^^ | ^^ |
+| MacBookPro5,2 | ^^ | hvds | ^^ |
+| MacBookPro5,3 | <span style="color:#30BCD5"> YES </span> | kommtzeitkonrad | - Stock model <br/>- Patcher version 0.0.22 |
```

Comment:
`<span style="color:#30BCD5"> YES </span>` should be replaced with `^^` to have proper formatting in the with the site